modules/ztp-image-based-upgrade-without-backup.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrading-without-backup_{context}"]
+= Upgrading the {sno} clusters with {lcao} without backup
+
+When you have generated the seed image and have completed the `Prep` stage, you can begin upgrading the target clusters.
+During the upgrade process, the OADP Operator creates a backup of the cluster, then {lcao} reimages it.
```

Comment:
It creates backups of the artifacts specified in the OADP backup CRs, it is not a backup of the full cluster